.github/workflows/desmos-bindings.yml

```diff
@@ -148,3 +143,62 @@ jobs:
         if: env.GIT_DIFF
         working-directory: ./packages/bindings
         run: cargo build --no-default-features --features reports --target wasm32-unknown-unknown
+
+      - name: Build all feature 🧪
+        if: env.GIT_DIFF
+        working-directory: ./packages/bindings
+        run: | 
+          cargo build --all-features --target wasm32-unknown-unknown
+
+  check:
```

Comment:
This step is to check the desmos-bindings wasm file has f64 or f32 operation or not.